### DISCUSSION TITLE
Running on cluster

### DISCUSSION THREAD

**Discussion by alib-l** [deepmodeling/deepmd-kit#469] [Q&A]
Hello, 
I want to use deepmd on a cluster. It seems "dp train" uses all cores which is not possible in a HPC. 
How can I specify the number of cores for training?

**Comment by njzjz** [comment]
Use environmental variables: `OMP_NUM_THREADS`, `TF_INTRA_OP_PARALLELISM_THREADS` and `TF_INTER_OP_PARALLELISM_THREADS`.
See https://github.com/deepmodeling/deepmd-kit/blob/v1.3.3/doc/use-deepmd-kit.md#training.

**Reply by alib-l** [reply]
Thank you so much
